terraform/modules/data-share-service/ecs-security.tf

```diff
@@ -13,9 +13,9 @@ resource "aws_security_group" "ecs_tasks" {
   }
 
   egress {
```

Comment:
Is this just to outside the VPC? For example we need 5432 for postgres - will that work?